scania-digital-design-system/sdds
# fix(banner): enabled cancelable

**Describe pull-request**  
Enabled cancelable events for banner. Also introduces sddsShow event.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1085

**How to test**  
1. Go to storybook link below.
2. Check in Components -> Banner -> Web Component
3. Check the console for the logging of events.

**Suggested test steps**
